wasm2c/README.md

```diff
@@ -395,3 +461,118 @@ $ wat-desugar fac-flat.wat --fold -o fac-folded.wat
 
 The formatting is different and the variable and function names are gone, but
 the structure is the same.
+
+## Instantiate multiple instances of a module
```

Comment:
How about `Create multiple instances of a module` or `Instantiating a module multiple times`?

Reply:
Done ("Create multiple instances of a module")